libmscore/repeatlist.cpp

```diff
@@ -260,19 +284,49 @@ void RepeatList::unwind()
             return;
 
 // qDebug("unwind===================");
+
+      for (Measure* m = fm; m; m = m->nextMeasure())
+            m->setPlaybackCount(0);
+
+      // partition score by section breaks, and unwind individual sections seperately
```

Comment:
Wait.  Regarding what @MarcSabatella said to me yesterday in the chat, a frame is not always a measure.  So I have a small bug when I partition score by section breaks here if a section break occures on a non-measure measure-base, for example the horizontal frame here:
![section_break_on_non-measure_frame](https://cloud.githubusercontent.com/assets/6502474/9330764/45efc6a8-4589-11e5-83bd-22047c673af5.png)

All this bug does is ignore the section break entirely (not a crash or anything).  So let me fix it here.

I also notice that a section-break on a frame will not incur a pause during playback.  That is a seperate bug I will file in musescore issue tracker.